L1Trigger/L1TMuon/plugins/L1TMuonProducer.cc

```diff
@@ -20,7 +20,7 @@
 
 // system include files
 #include <memory>
-#include <fstream>
+#include <sstream>
```

Comment:
This should stay fstream. sstream is not needed but fstream is needed for std::ofstream m_debugOut; in line 120.